gossip.go

```diff
@@ -81,11 +81,21 @@ func incIdx(tx *buntdb.Tx, m *Message) (index string, err error) {
 
 	if m != nil {
 		var b []byte
+
+		// fmt.Printf("\nHC: gossip.go: incIdx: 85: Message before encoding is:\n  %v\n", m)
+
 		b, err = ByteEncoder(m)
 		if err != nil {
 			return
 		}
 		msg = string(b)
+
+		// fmt.Printf("\nHC: gossip.go: incIdx: 93: Message after encoding is:\n  %v\n", msg)
+
+		var decodedMessage interface{}
```

Comment:
I think these two lines don't have any effect without the printfs?  In which case shouldn't they be commented out or removed?

Reply:
because the output to the test mechanism does have any colour output, and the ability to "make" it have colour output is some kind of dark fu that I dont get
Or I am wrong

Reply:
i "think" this may be becuase the output file is "text" and the format that includes colour output is " binary", though this may be completly untrue

Reply:
very similar to the GOB issue with monotonic time, just different

Reply:
potentially some kind of "urlencoding" might work here?